protostar/cheatable_starknet/cheaters/expect_events_controller.py

```diff
@@ -85,24 +84,22 @@ def compare_expected_and_actual_results(self):
 def match_events(
     expected_events: list[Event], emitted_events: list[Event]
 ) -> EventMatchingResult:
-    expected_events_queue = deque(expected_events)
     event_matchings: list[EventMatching] = []
     for emitted_event in emitted_events:
-        if len(expected_events_queue) == 0 or not should_accept_event_matching(
-            expected_event=expected_events_queue[0], emitted_event=emitted_event
+        if len(expected_events) == 0 or not should_accept_event_matching(
+            expected_event=expected_events[0], emitted_event=emitted_event
         ):
             event_matchings.append(SkippedEventMatching(emitted_event=emitted_event))
         else:
             event_matchings.append(
                 AcceptedEventMatching(
                     emitted_event=emitted_event,
-                    expected_event=expected_events_queue[0],
+                    expected_event=expected_events[0],
                 )
             )
-            expected_events_queue.popleft()
+            expected_events.pop(0)
     failed_event_matchings = [
-        FailedEventMatching(expected_event)
-        for expected_event in list(expected_events_queue)
+        FailedEventMatching(expected_event) for expected_event in list(expected_events)
```

Comment:
`expected_events` is already a list

Reply:
I forgot to change it after removing the queue.

[remove unnecessary converting to list](https://github.com/software-mansion/protostar/pull/1412/commits/2c6f9de34f0d7eb00c54d18726b4ae3b9a64e070)_